aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -1928,17 +1898,17 @@ Tensor compute_T18(const Tensor& A) {
 
   auto Bs = at::native::_compute_linear_combination(As, bs);
 
+  // tmp buffer for these matrix products
+  auto tmp = As.select(0, 0);
   // compute A9
-  Bs.select(0, 3).add_(at::native::matmul(
-    // tmp buffer for this matrix product
-    As.select(0, 0),
+  Bs.select(0, 3).add_(at::matmul_out(
```

Comment:
I'm not sure what you're benchmarking so my observation might be wrong - but you replaced a call to `at::native::matmul` with a call to `at::matmul_out`. That will add a bit of overhead, since the `at::native::` call bypasses the dispatcher, while `at::matmul_out` hops through the dispatcher before eventually hitting `at::native::matmul_out`.

`at::matmul_out` might be the thing we want, since we care about the dispatcher :). But it wouldn't be surprising that this line adds a bit of framework overhead.

Reply:
I replaced `at::native::matmul` with `at::matmul` because of the "we care about the dispatcher" point.